### PR TITLE
Add doc for load multiple pest files and `include!` syntax.

### PR DESCRIPTION
**Pull request by huacnlee**
Ref:

- https://github.com/pest-parser/pest/pull/759
- https://github.com/pest-parser/pest/pull/758

### CODE REVIEW

**Review by CAD97 — CHANGES_REQUESTED** [review]
The unrelated changes (probably an ~~overly~~ opinionated format-on-save?) don't belong in this PR, and the ones that touch link anchors are objectively incorrect.

**Comment by huacnlee** [comment]
It's by Markdown formatter. I have updated.

File: src/grammars/syntax.md [hunk 1]
```diff
@@ -384,6 +384,33 @@ fstring = @{ "\"" ~ ... }
...
+`toml.pest`:
+
+```pest
+include!("base.pest")
+
+pair  = { key ~ WHITESPACE* ~ "=" ~ WHITESPACE* ~ value }
+key   = @{ identifier }
+value = { !NEWLINE ~ ANY }
+```
+
```
Comment (tomtau) [inline src/grammars/syntax.md:413]:
```suggestion
```

File: src/grammars/grammars.md [hunk 2]
```diff
@@ -36,6 +36,22 @@ exists during compilation. However, you can use `Rules` just like any other
...
+use pest::Parser;
+
+#[derive(Parser)]
+#[grammar = "parser/base.pest"]
+#[grammar = "parser/grammar.pest"]
+struct MyParser;
+```
+
+Then `pest` will generate a `Rules` enum that contains all the rules from both.
+This is useful if you have a base grammar that you want to extend in multiple.
```
Comment (tomtau) [inline src/grammars/grammars.md:53]:
```suggestion
This is useful if you have a base grammar that you want to extend in multiple grammars.
```

**Review by tomtau — APPROVED** [review]

**Pull request merged**